api/service/src/main/java/com/coreeng/supportbot/teams/AzureUsersFetcher.java

```diff
@@ -10,7 +10,7 @@
 import static java.util.Objects.requireNonNull;
 
 /**
- * Requires Azure permissions: GroupMember.Read.All, User.Read.All
+ * Requires Azure permissions: GroupMember.Read.All, User.ReadBasic.All
```

Comment:
Should this not be in a readme somewhere as pre-requsities. Prevents someone having to dig into the code to check what is required to deploy our software on azure

Reply:
nvm just seen it now